### PR TITLE
Some cleanups for posts and sidebar

### PR DESCRIPTION
**Pull request by DevBefell**
Fixed some of the latest post's dates as well as managed the consistency of the titles.
Also updated Account Support link
For consistency I changed the version titles to (Minecraft VERSION: UPDATE NAME) and the posts to (DATE-release-VERSION).
It was the old way Minecraft did their post (now its "is out on Java") and is now much more distinct what posts are for what versions.

### CODE REVIEW

**Comment by bigfoot547** [comment]
Sorry, university is acting up. I will try and look it over later today or tomorrow. thanks

File: _includes/sidebar.html [hunk 1]
```diff
@@ -1,16 +1,27 @@
...
   <div style="height:5px; display:block;"></div>
-  <a href="https://www.minecraft.net/about-legends"><img src="{{ site.baseurl }}/img/logo_legends.png" border="0"></a>
+  <a href="https://www.minecraft.net/about-legends"><img src="{{ site.baseurl }}/img/logo_legends.png"></a>
   <h2>Community links:</h2>
-  <a href="http://minecraftforum.net/">Minecraft&nbsp;Forums</a><br><a href="http://www.twitch.tv/directory/game/Minecraft">Minecraft on Twitch.tv</a><br><a href="https://minecraft.wiki">Minecraft Wiki</a>
+  <a href="http://minecraftforum.net/">Minecraft&nbsp;Forums</a><br>
+  <a href="http://www.twitch.tv/directory/game/Minecraft">Minecraft on Twitch.tv</a><br>
+  <a href="https://minecraft.wiki">Minecraft Wiki</a><br>
   <h2>OLauncher:</h2>
-  <a href="https://github.com/olauncher"><img src="{{ site.baseurl }}/img/olauncher_logo_150.png" border="0"></a>
```
Comment (bigfoot547) [inline _includes/sidebar.html]: Your removal of `border="0"` causes borders to appear:
<details>
<summary>image</summary>
OLauncher sidebar with borders
</details>

**Review by bigfoot547 — CHANGES_REQUESTED** [review]
borders

**Comment by DevBefell** [comment]
Added it back.

**Review by bigfoot547 — APPROVED** [review]
looks good

**Pull request merged**